src/app/stores/AuthStore.ts

```diff
@@ -1,12 +1,76 @@
 import { RootStore } from 'app/stores/RootStore';
 import { observable } from 'mobx';
+import { SignupFormModel, LoginFormModel } from 'app/forms';
 
 export class AuthStore {
 
 
    @observable state: 'loading'|'loggedin'|'unauthed'
+
+
+   @observable signupForm: SignupFormModel
+   @observable loginForm: LoginFormModel
+
    constructor (private rootStore: RootStore){
       this.state = 'unauthed'
+      console.log("OY")
+      this.signupForm = new SignupFormModel()
+      this.loginForm = new LoginFormModel()
+      this.loadUser()
+   }
+
+   loadUser = async () => {
+      this.state = 'loading'
+
+      try{
+         const response = await fetch('/api/user', {
+            method: 'GET',
+            headers: {
+               'Content-Type' : 'application/json'
```

Comment:
Does react have global interceptors like Angular? If possible, we can intercept every API request and add the content-type header to it

Reply:
we can add this in the future.

https://www.npmjs.com/package/fetch-intercept

for now I'll continue this way